lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -186,8 +205,15 @@
                     }
                 },
 
-                goBackToTopic: function () {
-                    Backbone.history.navigate('topics/' + this.topicID, {trigger: true});
+                cancelTeam: function (event) {
+                    event.preventDefault();
+                    var url;
+                    if (this.action === 'create') {
+                        url = 'topics/' + this.topicID;
+                    } else {
```

Comment:
In case of non-bools, specifying the options explicitly ("`else if (this.action === 'edit')`) makes the code easier to understand.